src/main/java/org/veupathdb/service/eda/us/service/MetricsService.java

```diff
@@ -25,6 +32,34 @@ public GetMetricsUserAnalysesByProjectIdResponse getMetricsUserAnalysesByProject
             throw new BadRequestException("Can't parse startDate '" + startDate + "' or endDate '" + endDate + "'. Correct format is: YYYY-MM-DD", e);
         }
     }
+
+    @Override
+    public GetMetricsUserAnalysesReportsByProjectIdResponse getMetricsUserAnalysesReportsByProjectId(String projectId, String reportMonth) {
```

Comment:
Is there documentation someplace that indicates that this endpoint is harvesting data written by another (scheduled) process, and providing access to it for humans?

Reply:
also, have you reviewed the generate API does to check that they tell the story of the API well?

Reply:
I guess late night Dan doesn't write documentation, that's a job for early afternoon Dan :)